CHANGELOG.unreleased.md

```diff
@@ -20,6 +20,7 @@
 - [REST API] Backup logs are now available at `/rest/v0/backups/logs`
 - [Plugin/auth-oidc] Fix empty user names when using default config [Forum#59587](https://xcp-ng.org/forum/post/59587)
 - [ESXI import] Fix failing imports when using non default datacenter name [Forum#59543](https://xcp-ng.org/forum/post/59543) PR [#6729](https://github.com/vatesfr/xen-orchestra/pull/6729)
+- [Backup] Fix \_BackupWorker consuming too much memory and being killed by system during full VM backup PR [#6732](https://github.com/vatesfr/xen-orchestra/pull/6732)
```

Comment:
Users don't know what is `_BackupWorker`, but they can understand *the backup worker*.

You also need to explicit that it impacts full backups on S3 only.